packages/cf.js/src/app/index.ts

```diff
@@ -13,6 +13,8 @@ import {
 import * as abi from "../utils/abi";
 import { CfNonce } from "../utils/nonce";
 
+const { keccak256 } = ethers.utils;
```

Comment:
If you're doing this let's just get rid of the `keccak256` export in `abi.ts`

Reply:
I did